docs/checks/checks.yaml

```diff
@@ -61,6 +73,10 @@ checks:
         Build from source.
   Branch-Protection:
     risk: High
+    tags: supply-chain, security, source code, code review
```

Comment:
source-code, code-review (with the -) for consistency with "supply-chain"?

Reply:
done.